src/plotrecipes.jl

```diff
@@ -97,7 +97,7 @@ end
     :yguide --> yguide
     :label --> ""
     z = map(_prepare_plots, dims(A))
-    parent(A), index(z)
+    parent(A), lookup(z)
```

Comment:
```suggestion
    parent(A), parent(lookup(z))
```
Lets not risk makie only handling Vector